MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```
@@ -20,13 +20,17 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onLoadError: Observer<String?>?
```

`onLoadError` could be renamed to `onLoadingStateChange` since it's used to hide errors as well (with `nil`).